assets/arc-0031/server/src/mock/users.db.ts

```diff
@@ -0,0 +1,11 @@
+import type { User } from '../types'
+
+const users: User[] = [
+  {
+    authAcc: 'REPLACE_WITH_ALGORAND_USER_ADDRESS',
+    challenge: '',
+    chainId: 'TestNet'
```

Comment:
Here we should store the genesishas

Reply:
Addressed in https://github.com/deanstef/ARCs/pull/4/commits/057fa7aa2cc37f11e5e0c4713e8a10f7f8bcd93f